src/Ionide.ProjInfo.ProjectSystem/WorkspacePeek.fs

```diff
@@ -59,15 +60,16 @@ module WorkspacePeek =
                             yield dirInfo
 
                             for s in dirInfo.GetDirectories() do
-                                if not (ignored s.Name)
-                                then yield! scanDirs s (lvl + 1)
+                                if not (ignored s.Name) then
+                                    yield! scanDirs s (lvl + 1)
                     }
 
                 scanDirs dirInfo 0 |> Array.ofSeq
 
             let getInfo (t, (f: FileInfo)) =
                 match t with
-                | UsefulFile.Sln ->
+                | UsefulFile.Sln
+                | UsefulFile.Slnf ->
```

Comment:
the internal libs parse slnf files to extract out the solutions and the list of projects implicitly, so we don't actually have to do anything differently here.